### PR TITLE
Tests and Sorting

### PR DESCRIPTION
**Pull request by nikkypyra**

### CODE REVIEW

File: src/components/Header.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import Header from './Header'
+import { MemoryRouter } from 'react-router-dom'
+
+test('renders header with logo', () => {
+  const { getByAltText } = render(
+    <MemoryRouter>
+      <Header />
+    </MemoryRouter>
```
Comment (bierlingm) [inline src/components/Header.test.js:10]: Do you need MemoryRouter here? I'm not sure.

**Review by bierlingm — APPROVED** [review]
All looking very good, couldn't find nearly anything :)

File: src/components/Header.test.js (hunk 1 above)
Comment (nikkypyra) [reply src/components/Header.test.js:10]: My logo is linked to the home page within my header. That's why I used it :)

Comment (bierlingm) [reply src/components/Header.test.js:10]: Ah, understood! Then you're all good to go... 👍

**Pull request merged**